### ISSUE TITLE
Remove SVG from the psconvert documentation

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#6988]
gs >=9.16 no longer supports SVG output, but the psconvert documentation still mentions the SVG format in many places. I think we should remove SVG from the documentation.

**Are you willing to help implement and maintain this feature?** Yes

**Comment by PaulWessel** [comment]
I agree, there is no going back here.  If you have time, please remote things but maybe leave the ability to do **-Ts** under the hood and not give an error but perhaps a warning?

**Comment by seisman** [comment]
> leave the ability to do **-Ts** under the hood and not give an error but perhaps a warning

I'm a little confused. If gs doesn't support SVG, there is no way to have a SVG file, so GMT should give an error, right.

**Comment by PaulWessel** [comment]
OK, remove it all.
